src/main/java/net/moonlightflower/wc3libs/port/win/WinGameExeFinder.java

```diff
@@ -93,22 +94,17 @@ public static File fromDirIgnoreVersion(@Nonnull File dir, @Nonnull Orient.WinAr
                 break;
         }
 
-        List<File> relativeSearchPaths = Arrays.asList(WARCRAFT_III_EXE_PATH,
+        return Stream.of(WARCRAFT_III_EXE_PATH,
                 FROZEN_THRONE_EXE_PATH,
                 WAR3_EXE_PATH,
                 X86_EXE_PATH_131,
                 X64_EXE_PATH_131,
                 X86_EXE_PATH_132,
-                X64_EXE_PATH_132
-        );
-
-        for (File relativeSearchPath : relativeSearchPaths) {
-            File inDirPath = new File(dir, relativeSearchPath.toString());
-
-            if (inDirPath.exists()) return inDirPath;
-        }
-
-        throw new NotFoundException(new Exception("tried: " + relativeSearchPaths.toString()));
+                X64_EXE_PATH_132)
+            .map(relativePath -> new File(dir, relativePath.toString()))
```

Comment:
It's the same as before, but I would prefer the Path API instead of working with File+toString.